compute_transfer_examples/README.md

```diff
@@ -59,8 +59,8 @@ In the first example, the Compute and Transfer flow takes a user-provided source
 1. Edit `compute_transfer_examples/compute_transfer_example_1_definition.json` and replace the placeholder values:
    - `gcs_endpoint_id`: Your GCS Collection ID
    - `compute_endpoint_id`: Your Compute Endpoint ID
-   - `compute_func_id`: The UUID of the registered `do_tar` function
-   - `compute_transform_from` and `compute_transform_to`: If your GCS collection uses path mapping
+   - `compute_function_id`: The UUID of the registered `do_tar` function
+   - `compute_transform_from` and `compute_transform_to`: If your GCS collection uses [base path mapping](https://docs.globus.org/globus-connect-server/v5/data-access-guide/#configure_collection_base_path)
```

Comment:
The term "mapping" here collides with "mapped collections". I think the linked documentation simply refers to "base path", and I recommend using only that term here and dropping "mapping".